controller/contour/rbac.go

```diff
@@ -0,0 +1,372 @@
+package contour
```

Comment:
This needs the standard Contour copyright header.

Reply:
Same as https://github.com/projectcontour/contour-operator/pull/21#discussion_r498556284.